### DISCUSSION TITLE
Permission Definition Suggestions

### DISCUSSION THREAD

**Discussion by spospisil** [abpframework/abp#12414] [Ideas]
Was wondering if the PermissionDefinition class could have a property added that controls whether or not the created Permission is visible or not in the UI while still retaining it's features/functionality.  In other words I have an application that needs to add hundreds of PermissionDefinitions in order to utilize the ABP permissioning function([Authorize], etc) but don't want hundreds of permissions showing up in ABP's permission maintenance page.

**Comment by maliming** [comment]
hi

You can create an issue.
